webprojects/SUMO.md

```diff
@@ -0,0 +1,77 @@
+* [The SUMO l10n forum](https://support.mozilla.org/forums/l10n-forum). But first thing first, you need to [create a SUMO account](https://support.mozilla.org/users/authcontributor) in order to participate in the discussions in the forum.
+
+Localized strings in Pontoon will be synced up with sumo-l10n on GitHub every 20 minutes, making the repository the source of truth.
+
+Please ignore these legacy components:
+
+* LC_MESSAGES/buddyup.po contains strings for mobile-centric 1-to-1 support project.
+* LC_MESSAGES/yaocho.po contains strings for the mobile version of the support knowledge base.
+
+## Testing
+
+On Thursdays, SUMO developers perform synchronization so localized content is ready on staging for testing.
+
+On Mondays, SUMO developers perform synchronization so:
+
+* New strings from code and database edits go into sumo-l10n repository.
+* Translated strings from sumo-l10n are pushed to production.
+
+There are three ways to test the product:
+
+* Staging server: https://support.allizom.org​.
+* Production server: http://support.mozilla.org/.
+* Locally: if you want to test your update in a fast and independent way, you should [install your own local instance of Kitsune](https://kitsune.readthedocs.io/en/latest/localization.html#testing-localized-strings) on your computer.
+
+### Testing on production
+
+Localization updates are pushed on Thursdays on staging for review in context, and on Mondays on production. Either way, the SUMO team relies on you to find issues and report bugs that are usually fixed at a timely manner.
+
+### Testing locally
+
+According to SUMO developers, Kitsune is in a very fragile state. The SUMO team has very limited resources to support local installs. Unless you are technical and self sufficient, you are strongly advised to use Pontoon and staging server to preview your localization work.
+
+If you choose to give local install a try, follow the step by step instruction on [testing localized SUMO locally]((https://kitsune.readthedocs.io/en/latest/localization.html#testing-localized-strings), including local installation of Kitsune, file compilation and steps to commit changes to repository.
+
+## FAQ
+
+For any additional questions not covered here, you can contact [Michał](mailto:mdziewonski@mozilla.com).
+
+### How often are new strings added? Is there a sprint?
+
+No, there are no sprints. Strings are added at **as needed** basis. Whenever a release or change is required, new strings will be added to both Pontoon and Kitsune on Mondays.
+
+### How do I participate in the [SUMO l10n forum](https://support.mozilla.org/en-US/forums/l10n-forum)?
+
+To be part of the SUMO community, you need to [create a SUMO account](https://support.mozilla.org/en-US/users/authcontributor), which allows you to write in our community l10n forum and localize any articles you see in SUMO. This is separate from the account used to log into Pontoon in order to localize SUMO product and other Mozilla products and projects.
```

Comment:
drop our (styleguide…): in the community l10n forum
to localize SUMO and other… (drop product)

Reply:
I will update this part later in the day. need to head out now.

Reply:
argh... meant to say "your" not "our".  Rewrite both sentences.